core/src/main/java/hudson/Functions.java

```diff
@@ -759,9 +759,12 @@ public static boolean isMultiline(String s) {
     }
 
     /**
-     * Encodes a string into UTF-8 encoding
+     * Partially encodes a string into UTF-8 encoding. It uses the hudson.Util encode method to escape non-ASCII characters in URL.
```

Comment:
> hudson.Util encode

Irrelevant implementation detail?

Also, the output example has leading and trailing spaces not apparent in the input string.

Reply:
Hi, thank you for reviewing this PR.

I wanted to convey that this doesn't fully encode the whole string as UTF-8, it only encodes some of the characters. I am open to how better to word this so that it doesn't just read as 'encodes the whole string into utf-8'

Thank you for spotting the spaces in the example, I have done a commiit to change that.